src/components/ThemeSwitcher.jsx

```diff
@@ -0,0 +1,48 @@
+import Label from './Label';
+import Input from './Input';
+
+const Themes = [
+  { id: 'theme-choice-one', value: '1' },
+  { id: 'theme-choice-two', value: '2' },
+  { id: 'theme-choice-three', value: '3' },
+];
+
+const inputProps = [
+  {
+    className: 'checked checked:bg-themeOne-30 checked:hover:bg-themeOne-40',
+    id: 'theme-choice-one',
+    value: 'themeOne',
+    checked: true,
+  },
+  {
+    className: 'checked:bg-themeTwo-30 checked:hover:bg-themeTwo-40',
+    id: 'theme-choice-two',
+    value: 'themeTwo',
+  },
+  {
+    className: 'checked:bg-themeThree-30 checked:hover:bg-themeThree-40',
+    id: 'theme-choice-three',
+    value: 'themeThree',
+  },
+];
```

Comment:
Let's merge these two variables, also let's use lowercase to start variable names.